main.py

```diff
@@ -359,54 +359,61 @@ def performEvaluation(issue_data) -> Result:
     specimin_path = get_specimin_env_var()
     specimin_command = build_specimin_command(repo_name, os.path.join(issue_folder_abs_dir, issue_id), issue_data[JsonKeys.ROOT_DIR.value], issue_data[JsonKeys.TARGETS.value], qual_jar_dir if os.path.exists(qual_jar_dir) else "")
     
-    if specimin_path is not None and os.path.exists(specimin_path):
-        result = run_specimin(issue_id ,specimin_command, specimin_path)
-    else:
-        result = run_specimin(issue_id ,specimin_command, os.path.join(issue_folder_abs_dir, specimin_project_name))
+    if specimin_path is not None and not os.path.exists(specimin_path):
+        print("Clone copy of Specimin is used")
+        specimin_path = os.path.join(issue_folder_abs_dir, specimin_project_name)
+    
+    result = run_specimin(issue_id ,specimin_command, specimin_path)   
```

Comment:
This change seems unrelated to the main goal of this PR.

(I don't actually see a problem with this change, but it's generally good to keep separate changes separate. Is there a specific motivation to include this change here? If so, a comment in the source code about why you do it this way might be warranted.)